findspam.py

```diff
@@ -1118,7 +1441,12 @@ def toxic_check(post):
     return False, False, False, ""
 
 
-# noinspection PyUnusedLocal,PyMissingTypeHints
+if GlobalVars.perspective_key:  # don't bother if we don't have a key, since it's expensive
+    toxic_check = create_rule("toxic {} detected", func=toxic_check, whole_post=True, max_rep=101, max_score=2)
```

Comment:
Is the assignment to `toxic_check` used anywhere?

Reply:
Flake will complain at you for having an unused variable, though

Reply:
@ArtOfCode- No it does NOT at module level because module variables can be imported. Local variables are useless outside the function so flake8 complains about those.

Reply:
Huh, TIL